File: src/types/marketplace-install-result.ts
export type MarketplaceInstallTarget = "codex" | "claude" | "workspace";

export type MarketplaceInstallResult = {
  skillId: string;
  slug: string;
  target: MarketplaceInstallTarget | string;
  installedPath: string;
  fileCount: number;
};

export type MarketplaceUninstallResult = {
  removedPath: string;
  skillId: string | null;
};

File: src-tauri/src/models/marketplace_install_result.rs
use serde::Serialize;

#[derive(Debug, Serialize)]
#[serde(rename_all = "camelCase")]
pub struct MarketplaceInstallResult {
    pub skill_id: String,
    pub slug: String,
    pub target: String,
    pub installed_path: String,
    pub file_count: usize,
}
